components/derive/src/lib.rs

```diff
@@ -249,11 +249,52 @@ pub fn service_factory(input: TokenStream) -> TokenStream {
 ///
 /// Prefix of the `exonum` crate has two main values - `crate` or `exonum`. The default value
 /// is `exonum`.
+///
+/// ## `id_auto_increment`
+///
+/// ```text
+/// #[exonum_interface(id_auto_increment = true)]
```

Comment:
Nit: Doesn't `#[exonum_interface(id_auto_increment)]` work as well? I'd prefer this form here and in actual code.